lib/api/index.js

```diff
@@ -50,6 +50,8 @@ module.exports = {
   },
 
   TRACE_HTTP_HEADER: propagation.TRACE_HTTP_HEADER,
+  AMAZON_TRACE_HTTP_HEADER: propagation.AMAZON_TRACE_HTTP_HEADER,
+  REQUEST_ID_HEADER: propagation.REQUEST_ID_HEADER,
```

Comment:
think this one is still missing an `HTTP`?

```suggestion
  REQUEST_ID_HTTP_HEADER: propagation.REQUEST_ID_HTTP_HEADER,
```